files/en-us/web/javascript/guide/regular_expressions/index.html

```diff
@@ -278,7 +278,7 @@ <h2 id="Using_regular_expressions_in_JavaScript">Using regular expressions in Ja
 
 <p>The occurrences of <code>/d(b+)d/g</code> in the two statements are different regular expression objects and hence have different values for their <code>lastIndex</code> property. If you need to access the properties of a regular expression created with an object initializer, you should first assign it to a variable.</p>
 
-<h3 id="Advanced_searching_with_flags_2"><a name="Advanced_searching_with_flags">Advanced searching with flags</a></h3>
+<h3 id="Advanced_searching_with_flags_2">Advanced searching with flags</h3>
```

Comment:
```suggestion
<h3 id="advanced_searching_with_flags">Advanced searching with flags</h3>
```

Not sure why there's a `_2` here, but I don't think there should be 👍

Reply:
But yay for getting rid of any `<a>` tag inside a `<h2>` (or `<h3>`).

Reply:
> Not sure why there's a `_2` here, but I don't think there should be +1

My bet: The `name` of the inner `<a>` (now removed) was found first, thus, the wrapping `id` had to be assigned another ID to keep the HTML valid.

I removed the inner `<a>` after clicking `Fix fixable flaws`.